src/Compute/Compute/help/New-AzDiskConfig.md

```diff
@@ -343,6 +343,8 @@ Accept wildcard characters: False
 
 ### -ImageReference
 Specifies the image reference on a disk.
+The ID will be the ARM ID of the PIR or user image from which to create a disk.
```

Comment:
To modify the description of parameter, please update the help message in code implementation as well.
https://github.com/Azure/azure-powershell/blob/19a1390c80e9c2f86708b28b225a679dfa99cdc3/src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs#L138-L152